src/api/app/models/bs_request_actions_seen_by_user.rb

```diff
@@ -0,0 +1,24 @@
+class BsRequestActionsSeenByUser < ApplicationRecord
+  belongs_to :bs_request_action
+  belongs_to :user
+
+  after_create do |_record|
+    RabbitmqBus.send_to_bus('metrics', 'bs_request_action,seen=true count=1')
```

Comment:
This is enough information to know if someone is using this feature. It's not really useful to know exactly who has seen which specific request action to be able to tell that.